client/sources/ok_test/sqlite.py

```diff
@@ -140,11 +140,12 @@ def _import_sqlite(self):
         bool; True if able to import the sqlite3 module and the binding version
         is at least self.VERSION; False otherwise.
         """
-        try:
-            self.sqlite3 = importlib.import_module(self.MODULE)
-        except ImportError:
-            return False
-        return self.sqlite3.sqlite_version_info >= self.VERSION
+        return False # Hotfix to prevent segfaults (See Issue 140)
+        # try:
```

Comment:
Leaving here until we find a better fix (which might just be remove this entirely)